### PR TITLE
Fix people cycling per day -> cycle trips per day; fixes #261

### LINKED ISSUE #261: Labelling for trips per day
The interface refers in several places to 'People cycling per day', i.e. cyclists per day.

However, I don't think it's 'people' or 'cyclists' per day strictly, as a return trip would be two trips but one person/cyclist. This is a little confusing.

Can I suggest we change this from 'People cycling per day' to 'Cycle trips per day' throughout? I think that is unambiguous, and it also has the benefit of being slightly shorter.

### PR DESCRIPTION
**Pull request by mvl22**
Text changes as discussed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,7 +183,7 @@ <h2>Layer controls</h2>
 					<p>
 						<select name="colour" class="updatelayer" data-layer="rnet" aria-label="Route network colour">
 							<option value="none">None</option>
-							<option value="flow" selected>People cycling per day</option>
+							<option value="flow" selected>Cycle trips per day</option>
```
Comment (Robinlovelace) [inline index.html:186]: :+1:  to this, any chance of adding a tooltip?

**Review by Robinlovelace — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Robinlovelace) [reply index.html:186]: OK I see there is already this:

> The number of people cycling is the default view. It shows an estimate of the number of people cycling on each road for the selected trip purpose, network type, and scenario.

I suggest changing that to:

> The "Cycle trips per day" variable is the default view. This is an estimate of the average number of daily trips cycle trips in either direction passing through each segment, for the selected purpose, network type, and scenario.

**Comment by Robinlovelace** [comment]
If you can implement the above suggestion I think this is good to go. Any other improvements to the wording you can think of @wangzhao0217 or Cici (don't have the GH handle to hand)?

File: index.html (hunk 1 above)
Comment (mvl22) [reply index.html:186]: Updated. This is already within a tooltip.

**Comment by mvl22** [comment]
I've updated the text further to match the 'Suggested change' text above.

Ready for merging I think please.

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**